.travis.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     - travis_retry scripts/setup-tools-for-ubuntu.sh
     script:
     - make test
+    - travis_retry make -C enterprise-suite check-console-api
```

Comment:
Does it make sense to do travis_retry? Doesn't it mean it will fail multiple times in a row?

Also - can we rename the target to something a bit more intuitive. Like "check-console-api-monitors-in-sync" or something.

Reply:
The checking script downloads files from github, so it could fail due to network issues. I'll rename makefile targets to something better.

Reply:
The download should happen in a separate retry step in that case.